frontend/src/metabase-lib/v1/Question.ts

```diff
@@ -701,11 +711,10 @@ class Question {
   // Internal methods
   _serializeForUrl({
     includeOriginalCardId = true,
-    clean = true,
     includeDisplayIsLocked = false,
     creationType,
   } = {}) {
-    const query = clean ? Lib.dropEmptyStages(this.query()) : this.query();
```

Comment:
this should a no-op because everything is managed with MBQL lib; but parsing the query and running a MBQL lib method is a heavy operation which should be avoided